bytestring_splitter/__init__.py

```diff
@@ -159,26 +178,32 @@ def _parse_message_meta(message_type):
         except TypeError:
             message_class = message_type
 
+        seeker = 1
+
         try:
             # If a message length has been passed manually, it will be the second item.
             # It might be the class object VariableLengthBytestring, which we use
             # as a flag that this is a variable-length message.
-            message_length = message_type[1]
+            message_length = message_type[seeker]
+            if message_length == VariableLengthBytestring or int(message_type[seeker]):
```

Comment:
The `__len__` of `VariableLengthBytestring` is evaluated here?

Reply:
No, the inverse; it's possible that the `message_length` is `VariableLengthBytestring`.